<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QDialog" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>494</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>494</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget{background-color: rgb(255, 255, 255);}
QWidget#wid_image {background-color: rgb(0, 85, 255);}

QLabel#label_title { color:white; }

QLabel#label_logintips, QLabel#label_reg_passwd_tip, QLabel#label_reg_phone_tip, QLabel#label_reg_verify_tip { color:red; }

QCheckBox::indicator {
    width: 15px;
    height: 15px;
}

QCheckBox::indicator:unchecked {
    image: url(:/icon/rc/check_off.png);
}

QCheckBox::indicator:checked {
    image: url(:/icon/rc/check_on.png);
}

QLabel#label_barcode {border-image: url(:/image/rc/barcode.png);}

QPushButton#btn_login { border-radius: 4px;background-color: rgb(51, 123, 216); min-width: 80px; color:white; }
QPushButton#btn_login:pressed { border: 1px solid rgb(255, 170, 127); background-color:rgb(46, 0, 140); }

QPushButton#btn_reg_confirm { border-radius: 4px;background-color: rgb(51, 123, 216); min-width: 80px; color:white; }
QPushButton#btn_reg_confirm:pressed { border: 1px solid rgb(255, 170, 127); background-color: rgb(46, 0, 140); }

QPushButton#btn_verify_getcode { border-radius: 4px;background-color: rgb(51, 123, 216); min-width: 80px; color:white; }
QPushButton#btn_verify_getcode:pressed { border: 1px solid rgb(255, 170, 127); background-color: rgb(46, 0, 140); }

QPushButton#btn_reg_back { border-radius: 4px;background-color: lightgray; min-width: 80px; color:black; }
QPushButton#btn_reg_back:pressed { border: 1px solid rgb(255, 170, 127); background-color: darkgray; }

QPushButton#btn_close { border-image: url(:/icon/rc/close.png); }
QPushButton#btn_min { border-image: url(:/icon/rc/minium.png); }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="ImageForm" name="wid_image" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>240</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>240</height>
      </size>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_min">
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_close">
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_login">
         <widget class="LineEditForm" name="lineedit_usrname" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>201</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
         <widget class="LineEditForm" name="lineedit_passwd" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>201</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
         <widget class="QCheckBox" name="check_remberpasswd">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>107</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>记住密码</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_login">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>100</y>
            <width>84</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>登陆</string>
          </property>
         </widget>
         <widget class="LinkButton" name="btn_forgetpasswd">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>64</y>
            <width>84</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>忘记密码</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_barcode">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>20</y>
            <width>85</width>
            <height>85</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>85</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>85</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_barcode_info">
          <property name="geometry">
           <rect>
            <x>356</x>
            <y>110</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>扫一扫下载手机端</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_logintips">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>130</y>
            <width>311</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_register">
         <widget class="QPushButton" name="btn_reg_confirm">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>130</y>
            <width>91</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>提交获取</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_reg_back">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>130</y>
            <width>91</width>
            <height>28</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>返回登陆</string>
          </property>
         </widget>
         <widget class="LineEditForm" name="lineedit_reg_phone" native="true">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>10</y>
            <width>210</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>210</width>
            <height>30</height>
           </size>
          </property>
         </widget>
         <widget class="LineEditForm" name="lineedit_reg_passwd" native="true">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>47</y>
            <width>210</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>210</width>
            <height>30</height>
           </size>
          </property>
         </widget>
         <widget class="LineEditForm" name="lineedit_reg_verify" native="true">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>87</y>
            <width>211</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="label_getpasswd_info">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>10</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditForm</class>
   <extends>QWidget</extends>
   <header>uiclass.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LinkButton</class>
   <extends>QPushButton</extends>
   <header>uiclass.h</header>
  </customwidget>
  <customwidget>
   <class>ImageForm</class>
   <extends>QWidget</extends>
   <header>uiclass.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
